mgmtworker/mgmtworker/worker.py

```diff
@@ -42,14 +47,22 @@ def _setup_logger():
 def make_amqp_worker(args):
     operation_registry = ProcessRegistry()
     workflow_registry = ProcessRegistry()
+    operation_consumer = WaitingCloudifyOperationConsumer
+    workflow_consumer = WaitingCloudifyWorkflowConsumer
+    try:
```

Comment:
instead of this, i'd do inside of `wait_for_replication_mixin.py` something like

```python
try:
  import cloudify_premium
except ImportError:
  has_premium = False
else:
  has_premium = True
```
and check for that flag here
this way we'll only have one `try/except ImportError`, instead of two, and we'll lose the ugly import inside a func

Reply:
Fixed